modules/upgrade-49-acknowledgement.adoc

```diff
@@ -0,0 +1,15 @@
+
+// Module included in the following assemblies:
+//
+// * upgrading/rosa-cluster-upgrading-prepare.adoc
+// * upgrading/osd-cluster-upgrading-prepare.adoc
+
+:_content-type: CONCEPT
+[id="upgrade-49-acknowledgement_{context}"]
+= Administrator acknowledgment when upgrading to OpenShift 4.9
```

Comment:
I'm keen on suggesting that the Administrator acknowledgement might need to go before the prerequisites/procedure, since it's something that has to be done _before_ you can upgrade.